src/commands/createFunction/promptStepsV2/QuickPickInputStep.ts

```diff
@@ -0,0 +1,27 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IAzureQuickPickItem } from "@microsoft/vscode-azext-utils";
+import { localize } from "../../../localize";
+import { FunctionV2WizardContext } from "../FunctionV2WizardContext";
+import { PromptSchemaStepBase } from "./PromptSchemaStepBase";
+
+export abstract class QuickPickInputStep<T extends FunctionV2WizardContext> extends PromptSchemaStepBase<T> {
+    protected async promptAction<U>(context: T): Promise<U | undefined> {
+        const picks: IAzureQuickPickItem<U | undefined>[] = await this.getPicks<U>(context);
+
+        if (!this.input.required) {
+            picks.push({
```

Comment:
Can you utilize the skip for now constant here too?

Reply:
No, unfortunately the data types are different and I didn't think it was worth extracting it out as a constant. The QuickInputStep.ts needs to use a string, but the FunctionListStep.ts has an internal type called `TemplatePromptResult` that are specific strings that we are reading for.

It didn't feel like it was worth it to make a constant for the `Skip for now` due to that,.